index.js

```diff
@@ -123,7 +131,11 @@ function fromShortKey(key) {
     if (text === "--".repeat(4)) {
       throw new Error("Empty layers are not allowed");
     }
-
+    
+    if(!text.match(layerRegex)){
```

Comment:
Use `RegExp.protoype.test()` instead. It returns a boolean instead of an array.
```suggestion
    if (!layerRegex.test(text)) {
```